schemas/telemetry/health/health.4.schema.json

```diff
@@ -91,11 +91,25 @@
           "type": "object"
         },
         "pingDiscardedForSize": {
-          "maxProperties": 10,
-          "minProperties": 1,
           "properties": {
+            "<unknown>": {
```

Comment:
We include the observed keys as seen in query: https://sql.telemetry.mozilla.org/queries/72243/source